rm-datasource/src/main/java/io/seata/rm/datasource/DataCompareUtils.java

```diff
@@ -57,14 +55,6 @@ public static boolean isFieldEquals(Field f0, Field f1) {
                         if (f1.getValue() == null) {
                             return false;
                         } else {
-                            int f0Type = f0.getType();
```

Comment:
I want to temporarily keep fastjson, can you make a judgment?

Reply:
yes